test-requirements.txt

```diff
@@ -1 +1,2 @@
 odoo_test_helper
+odoo-addon-confirmation_wizard @ git+https://github.com/OCA/server-ux.git@refs/pull/942/head#subdirectory=setup/confirmation_wizard
```

Comment:
Seems that this dependency is already merged. It can be removed now, most people usually link the open PR in the description to give a quick overview on the status.

Also it would have been nice to keep it in a separate PR usually called [TMP] Add blablalb: Don't merge me